app/controllers/order/confirm_booking.js

```diff
@@ -20,6 +22,12 @@ export default Controller.extend(cancelOrderMixin, asyncTasksMixin, {
     );
   },
 
+  updateRequestedQuantityValue(record) {
```

Comment:
I can see same code repeated at two places is it because you are using the counter at two places ? If that's the case move the common code into mixin.

Reply:
Sure